tests/fixups/TraceFixup/main.cpp

```diff
@@ -10,6 +12,14 @@
 
 #include "Config.h"
 
+//////// Need to undefine Preprocessor definition for NONAMELESSUNION on this .cpp file only for this to work
+#define USETRACELOGGING
+#ifdef USETRACELOGGING
+#include <TraceLoggingProvider.h>
+#else

```

Comment:
Since you are defining USETRACELOGGING on line 16 this #else will never hit.  Is that intentional?

Reply:
As we seem to be good with the idea of tracelogging, the ifdefs will be removed.

Reply:
done.

Reply:
Thanks.